pagopa/io-app
# feat: [IOCOM-864,IOCOM-865] Update message attachment's preview with the new DS

## Short description
This PR updates the message attachment's preview screen to the new DS system.
| Preview  | Data Error | Preview Error |
| ------------- | ------------- | ------------- |
| ![Simulator Screenshot - iPhone 14 - 2024-01-23 at 18 54 24](https://github.com/pagopa/io-app/assets/5150343/198f1aaf-7b34-4567-a4ca-378eabd1bc1f) | ![Simulator Screenshot - iPhone 14 - 2024-01-23 at 18 53 46](https://github.com/pagopa/io-app/assets/5150343/0b228419-4389-4110-a57b-60bc20a762cc) | ![Simulator Screenshot - iPhone 14 - 2024-01-23 at 18 57 06](https://github.com/pagopa/io-app/assets/5150343/105ccf15-6738-4e45-ac7a-25c0a10056d6) |


## List of changes proposed in this pull request
- New screen and components, referencing the new design system's components only
- Removed in-preview attachment download

## How to test
Using the io-dev-api-server, download some attachment. After that, enable the new DS and navigate to previously downloaded attachments to see the new DS. Check both Android and iOS. Check both success and failure cases.
